i2c_bus: 1
i2c_slave_address: 21
i2c_registers: [0, 1, 2]
